pom.xml

```diff
@@ -188,18 +188,11 @@
         <dep.confluent.version>7.5.1</dep.confluent.version>
         <dep.docker.images.version>93</dep.docker.images.version>
         <dep.drift.version>1.21</dep.drift.version>
-<<<<<<< HEAD
         <dep.duct-tape.version>1.0.8</dep.duct-tape.version>
-        <dep.errorprone.version>2.23.0</dep.errorprone.version>
-        <dep.flyway.version>10.1.0</dep.flyway.version>
-        <dep.google.http.client.version>1.43.3</dep.google.http.client.version>
-        <dep.iceberg.version>1.4.2-SNAPSHOT</dep.iceberg.version>
-=======
+        <dep.iceberg.version>1.5.1-SNAPSHOT</dep.iceberg.version>
```

Comment:
Depending on a `-SNAPSHOT` makes our build non-deterministic (we will always grab the latest snapshot, not the same one each time). We should be careful with this and if there is a non-snapshot build available, we should consider using it.

Reply:
Checking with JC and Ben for a non-sbapshot version, though the 1.4 version with Ladou's fix also had a -SNAPSHOT version ...

Reply:
Confirmed with Ben, it is stable and deploy but there only is a -SNAPSHOT version.